consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,66 @@
-# Create your models here.
+import uuid
 from django.db import models
 
 
-class Question(models.Model):
+class UUIDPrimaryKeyBase(models.Model):
+    class Meta:
+        abstract = True
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+
+class TimeStampedModel(models.Model):
+    created_at = models.DateTimeField(editable=False, auto_now_add=True)
+    modified_at = models.DateTimeField(editable=False, auto_now=True)
+
+    class Meta:
+        abstract = True
+        ordering = ["created_at"]
+
+
+class Consultation(UUIDPrimaryKeyBase, TimeStampedModel):
+    name = models.CharField(max_length=256, blank=False)
+    slug = models.CharField(blank=False, null=False, max_length=256)
+
+
+class Section(UUIDPrimaryKeyBase, TimeStampedModel):
+    consultation = models.ForeignKey(Consultation, on_delete=models.CASCADE, null=True)
+    text = models.TextField(blank=True)
```

Comment:
I think this should be `name`?

Reply:
(And maybe not blank?)

Reply:
Now updated. Will need to dropdb again as recreated the migrations.